Gems/SceneProcessing/Code/Source/Config/Components/SceneProcessingConfigSystemComponent.cpp

```diff
@@ -39,23 +39,53 @@ namespace AZ
-            m_softNames.push_back(aznew FileSoftNameSetting("_anim", PatternMatcher::MatchApproach::PostFix, "Ignore", false,
-                { FileSoftNameSetting::GraphType(SceneAPI::DataTypes::IAnimationData::TYPEINFO_Name()) }));
 
+            PopulateDefaultSoftNameSettings();
             m_UseCustomNormals = true;
         }
 
+        void SceneProcessingConfigSystemComponent::PopulateDefaultSoftNameSettings()
+        {
+            auto settingsRegistry = AZ::SettingsRegistry::Get();
+            if (settingsRegistry)
+            {
+                AddSoftNameSettingsFromSettingResitry<NodeSoftNameSetting>(settingsRegistry, AssetProcessorDefaultNodeSoftNameSettingsKey);
+                AddSoftNameSettingsFromSettingResitry<FileSoftNameSetting>(settingsRegistry, AssetProcessorDefaultFileSoftNameSettingsKey);
+            }
+
+            if (m_softNames.empty())
+            {
+                // Defaults in case there's no config setup
+                m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]_?1(_optimized)?$", SceneAPI::SceneCore::PatternMatcher::MatchApproach::Regex, "LODMesh1", true));
+                m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]_?2(_optimized)?$", SceneAPI::SceneCore::PatternMatcher::MatchApproach::Regex, "LODMesh2", true));
+                m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]_?3(_optimized)?$", SceneAPI::SceneCore::PatternMatcher::MatchApproach::Regex, "LODMesh3", true));
+                m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]_?4(_optimized)?$", SceneAPI::SceneCore::PatternMatcher::MatchApproach::Regex, "LODMesh4", true));
+                m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Ll][Oo][Dd]_?5(_optimized)?$", SceneAPI::SceneCore::PatternMatcher::MatchApproach::Regex, "LODMesh5", true));
+                m_softNames.push_back(aznew NodeSoftNameSetting("^.*_[Pp][Hh][Yy][Ss](_optimized)?$", SceneAPI::SceneCore::PatternMatcher::MatchApproach::Regex, "PhysicsMesh", true));
+                m_softNames.push_back(aznew NodeSoftNameSetting("_ignore", SceneAPI::SceneCore::PatternMatcher::MatchApproach::PostFix, "Ignore", false));
+                // If the filename ends with "_anim" this will mark all nodes as "Ignore" unless they're derived from IAnimationData. This will
+                // cause only animations to be exported from the source scene file even if there's other data available.
+                m_softNames.push_back(aznew FileSoftNameSetting("_anim", SceneAPI::SceneCore::PatternMatcher::MatchApproach::PostFix, "Ignore", false,
+                    { FileSoftNameSetting::GraphType(SceneAPI::DataTypes::IAnimationData::TYPEINFO_Name()) }));
+            }
+        }
+
+        template<class SoftNameSettingsType>
+        void SceneProcessingConfigSystemComponent::AddSoftNameSettingsFromSettingResitry(
```

Comment:
repeat: Registry is misspelled.
```suggestion
        void SceneProcessingConfigSystemComponent::AddSoftNameSettingsFromSettingRegistry(
```